sympy/printing/pycode.py

```diff
@@ -134,17 +134,39 @@ def _print_Piecewise(self, expr):
         for arg in expr.args:
             e = arg.expr
             c = arg.cond
-            result.append('((')
+            if i == 0:
+                result.append('(')
+            result.append('(')
             result.append(self._print(e))
-            result.append(') if (')
+            result.append(')')
+            result.append(' if ')
             result.append(self._print(c))
-            result.append(') else (')
+            result.append(' else ')
             i += 1
         result = result[:-1]
-        result.append(') else None)')
-        result.append(')'*(2*i - 2))
+        if result[-1] == 'True':
+            result = result[:-2]
+            result.append(')')
+        else:
+            result.append(' else None)')
         return ''.join(result)
 
+    def _print_Relational(self, expr):
+        "Relational printer for Equality and Unequality"
```

Comment:
Internal methods (those with a name beginning with underscore) don't need a docstring, but I guess it doesn't hurt.